compile_win32

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
AFAIK we have a [`Dockerfile`](https://github.com/Koihik/LuaFormatter/blob/master/Dockerfile), although it might be little bit outdated, why not fix it?